examples/sftp-service-receive-file/sftp_service_receive_file.metatags

```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
+description: This example demonstrates receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
```

Comment:
Here also the description can be improved

Reply:
Fixed